dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumActionScope.kt

```diff
@@ -201,6 +201,7 @@ internal class RumActionScope(
                 id = context.sessionId,
                 type = ActionEvent.ActionEventSessionType.USER
             ),
+            source = ActionEvent.Source.ANDROID,
```

Comment:
We should be using the value defined in `CoreFeature.sourceName` instead, in order to be compatible with cross platform products

Reply:
I was thinking that I will do this change once we get there as this is part of the ReactNative related ticket. The `CoreFeature.sourceName` will have to be evaluated to those enums but I guess I can solve both things in this PR.